bpf/cpu/cpu.bpf.c

```diff
@@ -222,7 +221,7 @@ static int find_offset_for_pc(__u32 index, void *data)
   } 
   
   // Debug logs.
-  // bpf_printk("\t<- fetched PC %llx, target PC %llx (iteration %d/%d, left:%d, right:%d)", ctx->table->rows[mid].pc, ctx->pc, index, MAX_BINARY_SEARCH_DEPTH, ctx->left, ctx->right);
+  // bpf_printk("\t<- fetched PC %llx, target PC %llx (iteration %d/%d, mid: --, left:%d, right:%d)", ctx->table->rows[mid].pc, ctx->pc, index, MAX_BINARY_SEARCH_DEPTH, ctx->left, ctx->right);
```

Comment:
This is not a typo, just printing `--` as printing the middle index would be very confusing after modifying the left and right indices, but not the middle one until the next iteration